src/config/.npmpackagejsonlintrc.json

```diff
@@ -28,19 +29,26 @@
       "require-publishConfig": "off",
       "require-repository": "error",
       "require-scripts": "error",
+      "require-types": "off",
+      "require-typings": "off",
       "require-version": "error",
       "bin-type": "error",
       "files-type": "error",
       "version-type": "error",
       "valid-values-engines": ["error", [
         {
-          "node": ">=10.0.0",
+          "node": ">=12.0.0",
```

Comment:
We only support [LTS and Current](https://github.com/ipfs/community/blob/master/CONTRIBUTING_JS.md#supported-versions) so this should be `>=14.0.0`